Source/Protocols/Accurate.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  CellHeightGettable.swift
+//  ReactiveDataDisplayManager
+//
+//  Created by Alexander Filimonov on 02/03/2020.
+//  Copyright © 2020 Александр Кравченков. All rights reserved.
+//
+
+import Foundation
+
+/// Protocol for calculating height of element according to the model
+public protocol AccurateHeight: Configurable {
```

Comment:
Chosen naming seems a little bit strange imho. May be you will consider choosing variants like `HeightCalculatable` and `WidthCalculatable`?

Reply:
And what about generator's names?